src/uct/sm/mm/base/mm_iface.h

```diff
@@ -236,17 +236,19 @@ typedef struct uct_mm_iface {
  */
 #define UCT_MM_TL_DEFINE(_name, _md_ops, _rkey_unpack, _rkey_release, \
                          _cfg_prefix, _cfg_table) \
-    \
+    static void uct_component_##_name##_ctor(void); \
```

Comment:
seems it's already declared in UCT_TL_INIT, do we really need it here as well?
are UCT_MM_COMPONENT_DEFINE/UCT_TL_REGISTER_DEF using it?